src/composables/use-browser-detection.js

```diff
@@ -7,9 +7,18 @@ export const useBrowserDetection = () => {
 
 export const useBrowserIsBlink = () => {
   const browser = useBrowserDetection()
-  return browser.isChrome || browser.isEdge || browser.isOpera
+  if (browser !== null) {
+    return browser.isChrome || browser.isEdge || browser.isOpera
+  } else {
+    return false
+  }
 }
 
 export const useBrowserIsMobile = () => {
-  return useBrowserDetection().isMobile
+  const mobile = useBrowserDetection()
```

Comment:
thanks for the suggestion however I would say more beginners like myself would find the if-else statements easier to understand rather than ternary operators. if there are some optimization issues I will do it.

Yes for sure the `mobile` variable was just a placeholder for the logic. I will change it to `browser` .